docs/share_usage.md

```diff
@@ -0,0 +1,299 @@
+
+If enough space is not there or any 3PAR limit is hit, the status of share is 
+set to 'FAILED' along with appropriate error message which can be seen while 
+inspecting the share details.
+
+**Note:** 
+1. ‘size’ can be specified to override the default share size of 1TiB.  
+2. The FPG must have enough capacity to accommodate the share.
+
+### Mounting a share
+```
+docker run -it --rm  --mount src=<share-name>,dst=</mount-dir>,volume-driver=hpe --name <container-name> alpine /bin/sh
+```
+
+This command allows mounting of share 'share-name' inside the container 'container-name' on mount 
+directory 'mount-dir' using alpine image. A share can be mounted multiple times
+on the same host or different hosts that have access to the share. A share that
+is mounted multiple times on a host is unmounted only after the last container 
+mounting it is exited or stopped.
+
+Permissions if present are applied after mounting the share.
+
+### Un-mounting a share
+If container shell prompt is there, simply type 'exit' to unmount the share.
+If container is in detached mode, then retrieve container ID using 
+```docker ps -a``` and simply type:
+```
+docker stop <container-id>
+```
+
+### Inspecting a share
+```
+docker volume inspect <share-name>
+```
+Displays details of the share being inspected
+
+### Listing shares
+```
+docker volume ls
+```
+Lists all the shares
+
+### Removing a share
+```
+docker volume rm <share-name>
+```
+This command allows removing a share. If the share being removed happens to be
+the last share under its parent FPG, then the parent FPG is also removed.
+Please note that removal of parent FPG happens asynchronously on a child thread.
+
+### Displaying help
+```  
+docker volume create -d hpe -o filePersona –o help  
+```  
+This command displays help content of the file command with possible options that 
+can be used with it.
+
+### Displaying backend initialization status
+```  
+docker volume create -d hpe -o filePersona –o help=backends
+```  
+This command displays the initialization status of all the backends that have 
+been configured for file driver.
+
+## Known behavior / issues
+1. All the operations must be performed sequentially. E.g. concurrent creation 
+   of multiple shares can lead to ETCD lock failures.
+2. When block related configuration parameters are used inadvertently in file 
+   driver configuration or vice-versa, it does not result in any error - the
+   plugin simply ignores it.
+3. When both 'DEFAULT' and 'DEFAULT_BLOCK' backends are defined as block driver,
+   'DEFAULT_BLOCK' is not treated as a special keyword. Rather it becomes like 
+   any other backend defined in a multi-backend configuration. Same goes when 
+   'DEFAULT' and 'DEFAULT_FILE' are defined as file driver.
+4. When two backend sections are identically defined, even then each backend 
+   is treated differently and results in having their individual default FPGs
```

Comment:
Unsupported options are provided in hpe.conf this will not have any impact on the Plugin Behavior 
for Eg: snapcpg is not supported parameter for the FILE Configuration, but it is valid parameter for BLOCK configurations.

Reply:
#605

Reply:
Hi @prablr79 - bullet number 2 is about this only... I'll add the snapcpg example to it... Besides that, do you want me to add/remove/rephrase it?